### ISSUE TITLE
Milliways by Max Fog c64 dual drives gives disk error

### ISSUE THREAD

**Issue by kpeamon** [johanberntsson/ozmoo#62]
I tried to use ozmoo.online to create a two-disk version of Milliways for C64 but the interpreter says disk error. I can't guarantee that it isn't a VICE emulator issue. It worked well with VICE C128 mode and a d71-file.

**Comment by fredrikr** [comment]
Thanks for your report.

If you're using 1541 disks, you will need dual disk drives of course, *or* a single drive and an REU. Have you paid close attention to which drive you need to insert a disk into?

**Comment by kpeamon** [comment]
I was asked to put disk 2 in drive 9 which I did (VICE emulator) but I still got the disk error.

**Comment by kpeamon** [comment]
You are absolutely right, I did not realize that there were drive settings for each drive, I thought they were common. So drive 9 was disabled all the time :/   sorry and thanks for the advice :D
